torchao/sparsity/README.md

```diff
@@ -18,6 +18,37 @@ More concretely, we hope to provide tutorials and APIs for both sparse kernels (
 2. Recover accuracy loss of pruned model with custom pruning algorthim.
 3. Accelerate masked/pruned models on sparsity-supported hardware to realize performance improvements.
 
+## Success Stories
+
+#### segment-anything
+We applied 2:4 sparsity to accelerate segment-anything, as part of [segment-anything-fast](https://github.com/pytorch-labs/segment-anything-fast).
+The results mentioned in the REAADME of the repo compose sparsity with a suite of other inference acceleration techniques.
+
+From our benchmarking, we see a 1.1x speedup when running with SEGMENT_ANYTHING_FAST_USE_FLASH_4 enabled.
+
+```
+python benchmarks/benchmark_sam.py
+
+   block_only  batchsize           dtype  compile                  qkv                 proj                 lin1                 lin2         time     memory      img/s
```

Comment:
Can you format this as a table it's a bit hard to read because I have to scroll all the way to the right? Also what does  lin1 and lin2 mean? Presumably memory is in GB?

Also I feel like a quick sentence before this table along the lines of we have support for `SparseSemiStructuredTensor` we can apply it to each of the qkv of attention or just the proj and here's how to apply it would be helpful